conf/waivers

```diff
@@ -0,0 +1,98 @@
+/hardening/oscap(/with-gui)?/[^/]+/configure_bashrc_tmux
+/hardening/oscap(/with-gui)?/stig(_gui)?/postfix_prevent_unrestricted_relay
+    rhel == 8
+
+# rule ordering issue - accounts_password_pam_retry is checked first and passes,
+# and a later enable_authselect remediation breaks it
+# - see https://github.com/OpenSCAP/openscap/issues/1880
+/hardening/.+/accounts_password_pam_retry
+    rhel >= 8
+
+# https://github.com/ComplianceAsCode/content/pull/10499
+# these are missing the required partitions, making the oscap Anaconda addon
+# error out during remediaton
+/hardening/anaconda/cis_(server|workstation)_l1
+    status == 'error' and note.startswith('RuntimeError: installation failed')
+
+# bz1828871, won't be fixed on rhel7
+# needs to be remediated more than once due to rule ordering issues
+/hardening/anaconda(/with-gui)?/[^/]+/postfix_network_listening_disabled
+    rhel == 7
+
+# bz1778661 abrt package
+# - seems to be less predictable on new upstream content
+/hardening/anaconda(/with-gui)?/e8/rpm_verify_(ownership|permissions)
+    Match(rhel == 7, sometimes=True)
+
+# bz1825810 or maybe bz1929805
+# can be reproduced mostly reliably (95%) both via anaconda and oscap CLI,
+# but apparently we don't really care about old releases
+/hardening/[^/]+/with-gui/stig_gui/sysctl_net_ipv4_ip_forward
+    Match(rhel == 7, sometimes=True)
+
+# https://github.com/ComplianceAsCode/content/issues/10424
+/hardening/anaconda(/with-gui)?/[^/]+/service_nftables_disabled
+    ssg >= '0.1.68'
+
+# caused by us not caring whether the VM has it installed,
+# - remediation should fix it, but doesn't -- possibly caused by
+#   https://github.com/RHSecurityCompliance/contest/issues/15
+/hardening/oscap(/with-gui)?/[^/]+/package_scap-security-guide_installed
+    ssg >= '0.1.68'
+
+# TODO: something new? .. RHEL-8 on e8 and ism_o, RHEL-7 e8
+#  - seems to not happen on latest 8.9 nightlies ??
+#  - on latest 7.9, but upstream 2023/05 content
+/hardening/oscap/[^/]+/package_rear_installed
+    Match(rhel <= 8, sometimes=True)
+
+# TODO: GUI related
+#
+# happens normally only on rhel-7 e8, but it's more widespread with GUI,
+/hardening/[^/]+/with-gui/[^/]+/rpm_verify_(ownership|permissions)
+    rhel == 9
+/hardening/oscap/with-gui/e8/rpm_verify_(ownership|permissions)
+    rhel == 7
+# no idea, but appears on all RHELs
+/hardening/anaconda/with-gui/cis_workstation_l[12]/file_groupownership_sshd_private_key
+    ssg >= '0.1.68'
+# no clue
+/hardening/anaconda/with-gui/[^/]+/service_avahi-daemon_disabled
+    True
+# on RHEL-8 too
+/hardening/[^/]+/with-gui/cis_workstation_[^/]+/sysctl_net_ipv4_ip_forward
+    rhel == 8
+# also no clue, but not present in new upstream content
+/hardening/oscap/with-gui/pci-dss/smartcard_auth
+    rhel == 7 and ssg < '0.1.68'
+
+# TODO: new in 2023/05, on all CIS profiles, seems to be new rules
+/hardening/anaconda(/with-gui)?/cis[^/]*/firewalld_loopback_traffic_(restricted|trusted)
+    rhel == 9 and ssg >= '0.1.68'
```

Comment:
Very fresh rules. They weren't identified in productization yet. From brief look it's same issue as the first waiver there - not running `firewalld` in Anaconda. But let's wait for productization and upstream issue.

Reply:
Do we even tests CIS with GUI in productization? .. The rules seem old enough to be caught by last week's productization and I didn't see them fail there.

Reply:
Yes, CIS with GUI is tested in productization. It couldn't be caught by last week productization because both rules were added 2023-05-16 and and productization started 2023-05-13.

This week productization (finished this morning) should reveal it. I expect new upstream issues today/tomorrow.